### DISCUSSION TITLE
Bags within bags

### DISCUSSION THREAD

**Discussion by Cikoni-git** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#379] [Ideas]
Hi, it would be great if we could create actors which would behave as containers and we could put items in them. This is so we don't have to create whole characters if we wanted to just give players a chest or a bag to open and view.

**Comment by aMediocreDad** [comment]
Absolutely something I have considered. I'm currently, slowly, working on a rewrite of all the actor and item logic and so that takes precedence. But also certainly within the realm of possibility when I'm done with that work.
